relnotes/unregister.feature.md

```diff
@@ -0,0 +1,9 @@
+### Allow removing the client from the selected set in EpollDispatcher.unregister
+
+`EpollSelectDispatcher.unregister` now accepts an optional flag
+(`remove_from_selected_set`) which will remove the client from the selected set
+if it's in there. This will allow other `ISelectClient`s to safely destroy
+other `ISelectClient`s from their handle method (previously, this was unsafe
```

Comment:
Maybe reword the section in (). It's quite unclear.

(You could probably copy some of the doc from the `unregister` method, say.)